certbot/plugins/dns_common_lexicon.py

```diff
@@ -1,12 +1,23 @@
 """Common code for DNS Authenticator Plugins built on Lexicon."""
-
 import logging
 
 from requests.exceptions import HTTPError, RequestException
 
+# pylint: disable=unused-import, no-name-in-module
+from acme.magic_typing import Union, Dict, Any
```

Comment:
nit: Can we reenable these globally disabled pylint checks after this line. Syntax is `# pylint: enable=unused-import, no-name-in-module`.

Reply:
Corrected. I put it as inline comment, I saves two lines, and apply the disable only on this specific line.